src/main/java/xyz/nucleoid/fantasy/RuntimeWorldConfig.java

```diff
@@ -72,9 +72,10 @@ public RuntimeWorldConfig setGenerator(ChunkGenerator generator) {
         this.generator = generator;
         return this;
     }
-	
+
     public RuntimeWorldConfig setShouldTickTime(boolean shouldTickTime) {
         this.shouldTickTime = shouldTickTime;
+        this.gameRules.set(GameRules.DO_DAYLIGHT_CYCLE, shouldTickTime);
```

Comment:
Setting gamerule prevents the sun from moving on the client, and moving back on server sync (jitter)